ui-tests/tests/jupytereverywhere.spec.ts

```diff
@@ -458,19 +458,20 @@ test.describe('Landing page', () => {
 
 test.describe('Kernel Switching', () => {
   test('Should open kernel switcher menu', async ({ page }) => {
-    const downloadButton = page.locator('.je-KernelSwitcherButton');
-    await downloadButton.click();
+    const dropdownButton = page.locator('.je-KernelSwitcherButton');
+    await dropdownButton.click();
```

Comment:
This was just an incorrect variable name but I somehow noticed it now; happy to split it into its own PR if needed.